packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -3,7 +3,7 @@
         <lightning-card title={label}>
             <ul class="slds-has-dividers_around-space slds-m-bottom_medium">
                 <template if:true={hasParents}>
-                    <lightning-button variant="base" label="All" title="Show All" icon-name="utility:all"
+                    <lightning-button variant="base" label={labels.clear} title={labels.clear} icon-name="utility:all"
```

Comment:
I wasn't exactly sure what to do here because this all label and button are not part of the redesigned category facets. But "All" didn't really describe what this was doing either so I switched it to clear. I'll follow your suggestion and change the icon for clear.